experimental/packages/opentelemetry-instrumentation-fetch/test/fetch.test.ts

```diff
@@ -210,12 +210,21 @@ describe('fetch', () => {
     sinon.stub(core.otperformance, 'now').callsFake(() => fakeNow);
 
     function fakeFetch(input: RequestInfo | Request, init: RequestInit = {}) {
+      // Once upon a time, there was a bug (#2411), causing a `Request` object
+      // to be incorrectly passed to `fetch()` as the second argument
+      assert.ok(!(init instanceof Request));
+
       return new Promise((resolve, reject) => {
-        const response: any = {
-          args: {},
-          url: fileUrl,
-        };
```

Comment:
This variable is really the "default response body", but it also serves a double-duty as the `ResponseInit` (second argument to `new Response()`, which is very confusing. I separated  those into appropriately named separate variables.

To be clear, no tests actually cares that the mock "server response body" has the status in it.